src/lib/litegraph/src/types/widgets.ts

```diff
@@ -306,6 +306,12 @@ export interface IBaseWidget<
 
   hidden?: boolean
   advanced?: boolean
+  /**
+   * Set if the node is displayed on the parent subgraphNode
+   * Promoted widgets have a green border
+   * @readonly [Computed] This property is computed on graph change
+   */
```

Comment:
This is a lot of comment...
What happens if we change the theme color for the border here?